.github/workflows/make-test.yaml

```diff
@@ -11,10 +11,13 @@ jobs:
   build:
     # Pull request has label 'ok-to-test' or the author is a member of the organization
     if: contains(github.event.pull_request.labels.*.name, 'ok-to-test') || contains(fromJSON('["COLLABORATOR", "MEMBER", "OWNER"]'), github.event.pull_request.author_association)
+    strategy:
+      matrix:
+        k8s_version: [1.27.1,1.28.0,1.29.0]
```

Comment:
Maybe better to pickup last patch version?

Reply:
Well, I thought about it. My decision is due to the fact that the patch versions should be backwards compatible, hence it really does not matter which patch version is used in testing. Along with this, not all versions are available here https://storage.googleapis.com/kubebuilder-tools/ (where setup-envtest fetches assets).
Considering these two points, I decided to use the first patch version available, to not to track availability of the latest one.